packages/docs/src/guide/advanced/nested-sub-commands.md

```
@@ -0,0 +1,238 @@
+  name: 'git',
+  version: '1.0.0',
+  subCommands: {
+    remote: remoteCommand
+  }
+})
+```
+
+<!-- eslint-disable markdown/no-missing-label-refs -->
+
+> [!TIP]
+> The complete example code is [here](https://github.com/kazupon/gunshi/tree/main/playground/advanced/nested-sub-commands).
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)